target/arm/translate-a64.c

```diff
@@ -1392,6 +1410,16 @@ static void disas_comp_b_imm(DisasContext *s, uint32_t insn)
 
     gen_goto_tb(s, 0, s->pc);
     gen_set_label(label_match);
+
+#ifdef CONFIG_FLEXUS
+    FLEXUS_IF_IN_SIMULATION( gen_helper_flexus_insn_fetch_aa64( cpu_env,
```

Comment:
@neo-apz Should this line go before all the gen_goto_tb's? It's worth a test with this specific opcode to make sure that these callbacks fire.